CMSgov/mdct-mcr
# export refactor

## Description

Closes [MDCT-2193](https://qmacbis.atlassian.net/browse/MDCT-2193) and [MDCT-2195](https://qmacbis.atlassian.net/browse/MDCT-2195). 

Addressing a couple of tech debt items, increased code coverage, and implemented feedback from R&D. 

### How to test

- Generate an MCPAR report
- Export as PDF and take a look at it!
- Ensure that all required fields are rendered and any answered fields should populate

### Changed Dependencies

N/A

## Code author checklist

## Reviewer checklist (two different people)
